Translation/en/Text/regex.txt

```diff
@@ -27,11 +27,17 @@ r:"^([０-９0-9]+)月([０-９0-9]+)日$"=$1/$2
 r:"^付け毛([０-９0-9]+)$"=Hair $1
 r:"^付け毛([０-９0-9]+)（左）$"=Hair $1 (Left)
 r:"^付け毛([０-９0-9]+)（右）$"=Hair $1 (Right)
+r:"^スロット([０-９0-9]+) カラー([０-９0-9]+)$"=Slot $1 Color $2
+r:"^スロット([０-９0-9]+) 柄([０-９0-9]+) カラー$"=Slot $1 Pattern $2 Color
+sr:"^(?<location>パンスト|靴下|ボトムス|手袋|トップス|インナー\([上下]\)|靴|靴下) 柄(?<num_i>[０-９0-9]+) カラー$"=${location} Pattern ${num_i} Color
+sr:"^(?<location1>パンスト|靴下|ボトムス|手袋|トップス|インナー\([上下]\)|靴|靴下) カラー(?<num_j>[０-９0-9]+)$"= ${location1} Color ${num_j}
+sr:"^(?<location2>ネイル|アイライン|瞳|瞳孔)カラー(?<num_k>[０-９0-9]+)$"= ${location2} Color ${num_k}
```

Comment:
`_i` has a specific meaning (don't retranslate, use as as), `_j`, `_k` are meaningless.  Names don't have to be unique per file only per line


```suggestion
sr:"^(?<location>パンスト|靴下|ボトムス|手袋|トップス|インナー\([上下]\)|靴|靴下) カラー(?<num_i>[０-９0-9]+)$"= ${location} Color ${num_i}
sr:"^(?<location>ネイル|アイライン|瞳|瞳孔)カラー(?<num_i>[０-９0-9]+)$"= ${location} Color ${num_i}
```